tests/Microsoft.Fx.Portability.MetadataReader.Tests/TestAssembly.cs

```diff
@@ -49,6 +49,8 @@ public CSharpCompileAssemblyFile(string source, bool allowUnsafe)
 
             public string Version { get; }
 
+            public bool SkipBinaryIfPackageExists { get { return false; } }
```

Comment:
Please stay consistent with the rest of the file; `SkipBinaryIfPackagesExist => false;`

Reply:
Done